stable_baselines3/common/base_class.py

```diff
@@ -185,6 +185,11 @@ def __init__(
             if self.use_sde and not isinstance(self.action_space, gym.spaces.Box):
                 raise ValueError("generalized State-Dependent Exploration (gSDE) can only be used with continuous actions.")
 
+            if isinstance(self.action_space, gym.spaces.Box):
+                assert np.all(
+                    np.isfinite(np.array([self.action_space.low, self.action_space.high]))
+                ), "Continuous action space must have a finite lower and upper bound"
```

Comment:
Maybe a pointer to the doc about this (here? https://stable-baselines3.readthedocs.io/en/master/guide/rl_tips.html#tips-and-tricks-when-creating-a-custom-environment ). It is not exact answer but healthy tips how env should be made :).

Other than that LGTM!

Reply:
hmm, I think the error message is clear enough of what is wrong, and the env checker should do the rest ;)